src/app/clusters/identify-server/identify-server.cpp

```diff
@@ -37,13 +36,8 @@
 #error "identify requrires a device layer"
 #endif
 
-#ifndef emberAfIdentifyClusterPrintln
-#define emberAfIdentifyClusterPrintln(...) ChipLogProgress(Zcl, __VA_ARGS__);
-#endif
-
 using namespace chip;
 using namespace chip::app;
-using namespace chip::app::Clusters::Identify;
```

Comment:
You're right, this just was a mix up on my side. I was putting identify-server.h in the Identify namespace and that was causing the error. But since I stopped doing that, it wasn't necessary to remove this. I will fix it.